### ISSUE TITLE
Ability to disable `__worldX` and `__worldY` in json

### ISSUE THREAD

**Issue by Trouv** [deepnight/ldtk#1002]
Since I use version control with my LDtk project, and since my project treats levels as standalone with no geographical relation, I prefer it when changes to one level don't cause diffs in other levels. For this reason I have external levels enabled, and use a horizontal level layout.

However, with `__worldX` and `__worldY` now being stored on entities - changes to level size, or level order, or level addition/removal will cause all later levels to need to update these values for all of their entities. This actually doesn't happen every time, which might be a bug in itself, but when the other levels update they'll recalculate these values and have dirtier diffs than necessary. All of the levels might update due to something like #992, or due to a change in the default level background color.

For example, if I want to add a level in the middle of my game, I would want my git diff to have the new level metadata appear in the `.ldtk` asset, and then the new `.ldtkl` appear as a complete addition, and that's it. Instead, I might also get a bunch of changes to half of the level files that look like this...
```diff
@@ -380,7 +380,7 @@
                                        "__tags": [],
                                        "__tile": { "tilesetUid": 50, "x": 0, "y": 0, "w": 32, "h": 32 },
                                        "__smartColor": "#94D9B3",
-                                       "__worldX": 6976,
+                                       "__worldX": 7280,
                                        "__worldY": 288,
                                        "iid": "35e43400-dbe0-11ec-8458-159e6416e9ca",
                                        "width": 32,
@@ -396,7 +396,7 @@
                                        "__tags": [],
                                        "__tile": { "tilesetUid": 50, "x": 0, "y": 0, "w": 32, "h": 32 },
                                        "__smartColor": "#94D9B3",
-                                       "__worldX": 7168,
+                                       "__worldX": 7472,
                                        "__worldY": 96,
                                        "iid": "3623ae00-dbe0-11ec-8458-6740c4560845",
                                        "width": 32,
@@ -412,7 +412,7 @@
                                        "__tags": [],
                                        "__tile": { "tilesetUid": 50, "x": 0, "y": 0, "w": 32, "h": 32 },
                                        "__smartColor": "#94D9B3",
-                                       "__worldX": 7392,
+                                       "__worldX": 7696,
                                        "__worldY": 64,
                                        "iid": "373df7a0-dbe0-11ec-8458-0d1bf2ba26bc",
                                        "width": 32,
@@ -428,7 +428,7 @@
                                        "__tags": [],
                                        "__tile": { "tilesetUid": 50, "x": 0, "y": 0, "w": 32, "h": 32 },
                                        "__smartColor": "#94D9B3",
-                                       "__worldX": 7168,
+                                       "__worldX": 7472,
                                        "__worldY": 320,
                                        "iid": "37670370-dbe0-11ec-8458-ddd3aca59c5b",
                                        "width": 32,
```

I'd like to either have the option to disable these values entirely,  or maybe they could just be disabled for horizontal / vertical worlds.

**Comment by Madour** [comment]
I agree, and anyways we can get entity world pos by doing e.g (level.worldX + entity.px.x + layer.__pxTotalOffsetX) right ?

**Comment by deepnight** [comment]
You're right. _worldX/Y are irrelevant on Horizontal/Vertical layouts anyway 🤷

**Comment by deepnight** [comment]
Fixed in 1.5
